planning/behavior_velocity_planner/autoware_behavior_velocity_crosswalk_module/src/scene_crosswalk.cpp

```diff
@@ -881,7 +881,7 @@ void CrosswalkModule::applySlowDown(
   }
   if (slowdown_pose)
     planning_factor_interface_->add(
-      output.points, planner_data_->current_odometry->pose, *slowdown_pose, *slowdown_pose,
+      output.points, planner_data_->current_odometry->pose, *slowdown_pose,
```

Comment:
Maybe we have start & end point for slowdown.

Reply:
As far as I read the code, crosswalk module inserts one slow down start point, and all trajectory points are overwritten with same slow down velocity. Then, crosswalk has only one control point as well.